moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -509,7 +509,7 @@ RobotTrajectory& RobotTrajectory::setRobotTrajectoryMsg(const moveit::core::Robo
 void RobotTrajectory::findWayPointIndicesForDurationAfterStart(double duration, int& before, int& after,
                                                                double& blend) const
 {
-  if (duration < 0.0)
+  if (duration < 0.0 || waypoints_.empty())
```

Comment:
The change in line 539 is not enough to handle the case that waypoints is empty. i.e. index an empty deque with `[0]`